cmd/serve.go

```diff
@@ -41,6 +41,7 @@ var CmdServ = cli.Command{
 
 func setup(logPath string) {
 	setting.NewContext()
+	setting.NewService()
```

Comment:
in order to load in `struct Service` the values from the configuration file. In particular, I was looking for a way to get `RequireSignInView`. 

Looks like `newService()` is the function which load this information but, I want load just this values not all the initialization done by `NewServices()` so I think, get a public function just for this  is the right way to get it.